### PR TITLE
Vbet bl 2

### PR DESCRIPTION
**Pull request by Cashe93**
Does not include 'Drainage Network should use same as Hydrography in RS_Context' task

### CODE REVIEW

**Comment by joewheaton** [comment]
@Cashe93 since I can't view (in GitHub) the layer files (they are binaries) and I'm too lazy to download and test, would you mind showing me screen shots in your issue or pull requests of what the symbology looks like? It will make my review easier. Thanks!

File: RaveAddIn/XML/vbet.xml [hunk 1]
```diff
@@ -28,13 +30,12 @@
...
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='VBET_NETWORK']" type="vector" symbology="" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_50']" type="vector" symbology="threshold_50" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_60']" type="vector" symbology="threshold_60" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_70']" type="vector" symbology="threshold_70" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_80']" type="vector" symbology="threshold_80" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_90']" type="vector" symbology="threshold_90" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_100']" type="vector" symbology="threshold_100" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" />
```
Comment (joewheaton) [inline RaveAddIn/XML/vbet.xml:39]: Thanks for adding in symbology keys here. Please:
- [ ] ditch the `xpathlabel` and use labels that mean something (remember that these are [thresholed and vectorized](https://riverscapes.github.io/vbet/algorithm.html#step-5---thershold-and-vectorize), but "unsantized". Maybe these nodes have a parent node of `Unsantized Raw Valley Bottom Extent` and then calli indviduals - `50% Threshold Raw VBET`
- [ ] I think you'll want to make each of these transparent if they are a solid color, but maybe you should just make them hollow (in which case they don't need transparent)

File: RaveAddIn/XML/vbet.xml [hunk 2]
```diff
@@ -8,14 +8,16 @@
         <Children>
           <Node label="Drainage Network">
             <Children>
+
+<Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='VBET_NETWORK']" type="vector" symbology="" />
```
Comment (joewheaton) [inline RaveAddIn/XML/vbet.xml:12]: Please see my reply on #63 for what to do here.

File: RaveAddIn/XML/vbet.xml [hunk 3]
```diff
@@ -8,14 +8,16 @@
...
               <Node xpathlabel="Name" xpath="Realizations/VBET/Inputs/Vector[@id='FLOWLINES']" type="vector" symbology="flow_lines" />
               <Node xpathlabel="Name" xpath="Realizations/VBET/Inputs/Vector[@id='FLOW_AREA']" type="vector" symbology="flow_lines" />
             </Children>
           </Node>
           <Node label="Lines of Evidence">
             <Children>
               <Node label="Slope Analysis" xpath="Realizations/VBET/Inputs/Raster[@id='SLOPE_RASTER']" type="raster" symbology="slope" />
-              <Node xpathlabel="Name" xpath="Realizations/VBET/Inputs/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" />
+              <Node label="Height Above Nearest Drainage" xpath="Realizations/VBET/Inputs/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" />
```
Comment (joewheaton) [inline RaveAddIn/XML/vbet.xml:20]: Thank you! Good.

**Comment by Cashe93** [comment]
Hey @joewheaton here is a screenshot of the symbology for intermediate thresholds. 

vbet_int_threshholds

File: RaveAddIn/XML/vbet.xml (hunk 1 above)
Comment (Cashe93) [reply RaveAddIn/XML/vbet.xml:39]: Can do! @joewheaton

**Comment by philipbaileynar** [comment]
@tyler1218hatch I'm going to rebase this today and out out a rave release. Ok?

**Comment by tyler1218hatch** [comment]
Sounds good Phillip, thanks. I'm still getting used to the methods of requesting and resolving changes, so let me know if there's anything I can do next time to make this go smoother.

File: RaveAddIn/XML/vbet.xml (hunk 3 above)
Comment (philipbaileynar) [reply RaveAddIn/XML/vbet.xml]: Yes. This is the correct approach

**Comment by philipbaileynar** [comment]
This pull request was rebased locally by Philip.

**Pull request closed without merging**